src/engine/cuecontrol.cpp

```diff
@@ -102,6 +102,9 @@ CueControl::CueControl(QString group,
     m_pCueIndicator = new ControlIndicator(ConfigKey(group, "cue_indicator"));
     m_pPlayIndicator = new ControlIndicator(ConfigKey(group, "play_indicator"));
 
+    m_pSignalBeginPosition = new ControlObject(ConfigKey(group, "signal_begin_position"));
```

Comment:
along with renaming the cue type, please call these 

fade_in_cue_point
fade_out_cue_point

or similar?